bin/pic-configure

```diff
@@ -46,6 +46,7 @@ help()
     echo "-c | --cmake         - overwrite options for cmake"
     echo "                       (e.g.: \"-DPIC_VERBOSE=21 -DCMAKE_BUILD_TYPE=Debug\")"
     echo "-t <presetNumber>    - configure this preset from cmakeFlags"
+    echo "-f | --force         - clear the cmake file cache and scan for new param files"
```

Comment:
new [param] files is a bit of a lie, since it will scan for all new `include/` files, but it's ok with me since it is more clear to the user when they should use it.

Reply:
Yes I keep it more user-focused.